### PR TITLE
Shipping labels: Checkbox styles

### PR DESCRIPTION
**Pull request by 0nko**
This PR just updates the checkbox base style and fixes multi-line checkbox alignment in SL.

| image | image |
|-|-|

### CODE REVIEW

File: WooCommerce/src/main/res/layout/fragment_edit_shipping_label_payment.xml [hunk 1]
```diff
@@ -94,13 +94,15 @@
...
+                style="@style/Woo.CheckBox"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:layout_marginStart="@dimen/major_100"
                 android:layout_marginTop="@dimen/major_150"
                 android:layout_marginEnd="@dimen/major_100"
                 android:layout_marginBottom="@dimen/major_150"
                 android:gravity="top"
+                android:paddingTop="@dimen/minor_25"
                 android:paddingStart="@dimen/minor_100"
```
Comment (hichamboushaba) [inline WooCommerce/src/main/res/layout/fragment_edit_shipping_label_payment.xml]: I think we should get rid of the `paddingStart`, and keep using the one from the style.

File: WooCommerce/src/main/res/layout/view_shipping_label_order_summary.xml [hunk 2]
```diff
@@ -129,11 +129,14 @@
 
     <com.google.android.material.checkbox.MaterialCheckBox
         android:id="@+id/mark_order_complete_checkbox"
+        style="@style/Woo.CheckBox"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:layout_marginStart="@dimen/minor_100"
         android:layout_marginTop="@dimen/major_100"
         android:paddingStart="@dimen/major_75"
```
Comment (hichamboushaba) [inline WooCommerce/src/main/res/layout/view_shipping_label_order_summary.xml:137]: Same as above

**Review by hichamboushaba — COMMENTED** [review]
Looks good, I just left some comments regarding the padding.

File: WooCommerce/src/main/res/values/styles_base.xml [hunk 3]
```diff
@@ -523,7 +523,7 @@ theme across the entire app. Overridden versions should be added to the styles.x
         <item name="android:textAppearance">?attr/textAppearanceSubtitle1</item>
     </style>
 
-    <style name="Woo.CheckBox" parent="Widget.AppCompat.CompoundButton.CheckBox">
+    <style name="Woo.CheckBox" parent="Widget.MaterialComponents.CompoundButton.CheckBox">
         <item name="paddingStart">@dimen/major_100</item>
         <item name="android:paddingLeft">@dimen/major_100</item>
```
Comment (hichamboushaba) [inline WooCommerce/src/main/res/values/styles_base.xml]: We should find the correct value for `paddingStart`, I think `major_100` gives more padding than what we have in Figma, as the checkbox drawable has already some padding in Android. I think `major_75` gives the most accurate results to my eye. WDYT?

**Comment by AliSoftware** [comment]
Moving this PR's milestone to 6.3 as agreed in Slack, as I'm code-freezing 6.2 today

**Comment by 0nko** [comment]
Thanks @hichamboushaba. All paddings have been addressed.

File: WooCommerce/src/test/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/ShippingCarrierRatesViewModelTest.kt [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+        viewModel.event.observeForever { event = it }
+
+        viewModel.onPackageSelected(availablePackages.first())
+
+        assertThat(event).isInstanceOf(ExitWithResult::class.java)
+        val result = (event as ExitWithResult<ShippingPackageSelectorResult>).data
+        assertThat(result.position).isEqualTo(0)
+        assertThat(result.selectedPackage).isEqualTo(availablePackages[0])
+    }
+}
```
Comment (hichamboushaba) [inline WooCommerce/src/test/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/ShippingCarrierRatesViewModelTest.kt]: @0nko did you intend to include this file in this PR?

Comment (0nko) [reply WooCommerce/src/test/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/ShippingCarrierRatesViewModelTest.kt]: Oops.

**Review by hichamboushaba — APPROVED** [review]
:shipit:

**Pull request merged**